spot-ingest/common/kafka_client.py

```diff
@@ -1,7 +1,7 @@
 #!/bin/env python 
 import logging
 import os
-from oni.utils import Util
+from common.utils import Util
```

Comment:
This is fine for now, i think i'm fond of spot.utils to differentiate from other possible tools that may be added